addons/l10n_ar/models/res_partner.py

```diff
@@ -125,3 +125,9 @@ def _get_id_number_sanitize(self):
             id_number = re.sub('[^0-9]', '', self.vat)
             res = int(id_number)
         return res
+
```

Comment:
cancel this pr and create a new one on enterprise modifying _vat_book_get_partner_document_code_and_number() method.


```diff
-           return doc_code, doc_number.rjust(20, '0')
+          return doc_code, (doc_number or '').rjust(20, '0')
```